stdlib/contextlib.pyi

```diff
@@ -64,9 +64,13 @@ class AbstractAsyncContextManager(ABC, Protocol[_T_co, _ExitT_co]):  # type: ign
         self, exc_type: type[BaseException] | None, exc_value: BaseException | None, traceback: TracebackType | None, /
     ) -> _ExitT_co: ...
 
+class _WrappedCallable(Callable[_P, _R]):
```

Comment:
That was my bad. `Callable` is a special form, not a class, but the following could work:

```suggestion
class _WrappedCallable(Generic[_P, _R]):
    def __call__(self, *args: _P.args, **kwargs: _P.kwargs) -> _R: ...
```

Reply:
I'll switch back to `Generic` to show the `jax` failure that version reported in the `mypy_primer` comment (`mypy` doesn't seem to consider `_WrappedCallable[_P, _R]` as conforming to `_Callable[_P, _R]`)

Reply:
Although, thinking about it, the old signature here was genuinely *wrong*, since it claimed the returned callable type would be the *same* type as the input (and hence have any extra attributes, etc, that a callable subclass defined), rather than just returning a new callable with a compatible calling signature.

The newly introduced `mypy` complaints in `jax` reflect that it had passed along that same erroneous claim.